docker/scripts/java-15-vars.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+JAVA_INSTALL_VERSION_MAJOR='15'
+JAVA_INSTALL_VERSION=2
+JAVA_INSTALL_PATH=/usr/opt/java-${JAVA_INSTALL_VERSION_MAJOR}-${JAVA_INSTALL_VERSION}/
+JDK_FILE=/tmp/jdk-14.0.2_linux-x64_bin.tar.gz
```

Comment:
wrong version

Reply:
also need to add download command to get the zip from s3

Reply:
Oh snap. Thanks. will fix.

Reply:
Will fix after we discuss whether we want this or not.